tailwind.config.cjs

```diff
@@ -7,7 +7,7 @@ module.exports = {
       colors: {
         'primary': 'var(--primary-color, #2563eb)',
         'primary-light': 'var(--primary-color-light, #dbeafe)',
-        'primary-dark':  'var(--primary-color-dark, #1e3a8a)',
+        'primary-dark':  'var(--primary-color-dark, #dbeafe)',
```

Comment:
I think I made a change here in the wip commit because this is the same color as `primary-light`. So I updated this to use a darker shade from figma for 'primary-dark`.

Reply:
updated with micho's color! also made a react-theme pr